app/Livewire/Forms/EditionEventForm.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+namespace App\Livewire\Forms;
+
+use App\Models\EditionEvent;
+use Livewire\Attributes\Validate;
+use Livewire\Form;
+
+class EditionEventForm extends Form
+{
+    public $editionEvent;
+
+    #[Validate(['required', 'date'])]
+    public $start_at;
+
+    #[Validate(['required', 'date'])]
+    public $end_at;
```

Comment:
I can select dates in the past, is this a behaviour that is expected?

Reply:
Past dates from what? The events are indeed supposed to be happening before the actual execution of the edition.